charts/gadget/templates/configmap.yaml

```diff
@@ -17,3 +17,5 @@ data:
       crio-socketpath: {{ .Values.config.crioSocketPath }}
       docker-socketpath: {{ .Values.config.dockerSocketPath }}
       podman-socketpath: {{ .Values.config.podmanSocketPath }}
+      verify-gadgets: {{ .Values.config.verifyGadgets }}
```

Comment:
I see we have `verify-gadgets` and operator params (see below) doing the same thing. Is this expected?

```
    operator:
      oci:
        verify-image: false
```

I think operator param might be enough. What do you say?

Reply:
I see how that is the case for deploy CLI flags. But `operator.oci.verify-image` is equivalent to `verify-gadgets` since both configure OCI operator so we can have something like:

```
    operator:
      oci:
        verify-image: {{ .Values.config.verifyGadgets }}
```

Then we can avoid the if clauses as well. Am I missing something?

Reply:
I'm wondering how we should map the flags from "kubectl gadget deploy" to the operators' configuration. In this PR we're mapping verify-gadgets to operators.oci.verify-image, but IMO this approach won't scale as we'll keep adding more and more flags. Perhaps we can just continue and fix it on #3168 later on.

Reply:
> but IMO this approach won't scale as we'll keep adding more and more flags. 

By this approach you mean defining `verify-gadgets` or about directly defining `operator.oci.verify-image` in the configmap (like above)?

Reply:
This approach == defining verify-gadgets